src/main/java/com/microsoft/azure/functions/worker/Constants.java

```diff
@@ -6,4 +6,6 @@
 public final class Constants {
     private Constants(){}
     public final static String TRIGGER_METADATA_DOLLAR_REQUEST_KEY = "$request";
+    public final static String FUNCTIONS_WORKER_DIRECTORY = "FUNCTIONS_WORKER_DIRECTORY";
+    public final static String JAVA_REVERSE_LIB_LOADING = "JAVA_REVERSE_LIB_LOADING";
```

Comment:
Rename to : `FUNCTIONS_WORKER_JAVA_USE_SHADED_LIBRARIES` - to make this more explicit and is a java worker behavior

Reply:
We will use shaded jars regardless, this setting to load which jars first, worker or client.
What about FUNCTIONS_WORKER_JAVA_CLIENT_LIB_FIRST?

Reply:
`FUNCTIONS_WORKER_JAVA_LOAD_APP_LIBS`  -- this means that libs from app take precedence. No need to add First?

Reply:
Sounds good, I will change it, the reason I added first that in both cases we load the jars one take presence over the other.